edge/pkg/eventbus/event_bus.go

```diff
@@ -99,14 +101,16 @@ func (eb *eventbus) Start(c *context.Context) {
 		}
 
 		if qos.(int) < int(packet.QOSAtMostOnce) || qos.(int) > int(packet.QOSExactlyOnce) || sessionQueueSize.(int) <= 0 {
-			panic("mqtt.qos must be one of [0,1,2] or mqtt.session-queue-size must > 0")
+			log.LOGGER.Errorf("mqtt.qos must be one of [0,1,2] or mqtt.session-queue-size must > 0")
```

Comment:
The benefit of panic is the exception callstack. Here we log error in the module start where the error is clear. So log the error and exit is fine.